CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ These changes are available on the `master` branch, but have not yet been releas
 
 ### Added
 
+- Added function `delete_auto_moderation_rule`.
```

Comment:
```suggestion
- Added function `Guild.delete_auto_moderation_rule`.
```

Reply:
The entry should also be appended to the bottom of the "Added" section, not the top.